src/components/divider/index.stories.mdx

```diff
@@ -0,0 +1,31 @@
+import { Meta, Story, Canvas, ArgsTable } from "@storybook/addon-docs";
+
+import Divider from ".";
+
+<Meta title="Components/Divider" component={Divider} />
+
+# Divider
+
+--- Add a description of the component here ---
+
+## Usage
+
+```jsx
+import { Divider } from "@wpmedia/arc-themes-components";
+
+const Feature = () => <Divider>Component Children</Divider>;
+```
+
+## Properties
+
+<ArgsTable of={Divider} />
+
+## Stories
+
+** Divider **
+
+<Canvas>
+	<Story name="Divider">
+		<Divider>Divider Text</Divider>
```

Comment:
Also, no text here.

Reply:
Fixed [2c036af](https://github.com/WPMedia/arc-themes-components/pull/145/commits/2c036afe131097f7f3991c2ae7546efc2b527f7c)

Reply:
this has been addressed -- nice work @malavikakoppula